examples/components/progress_circular/basic_usage.html

```diff
@@ -0,0 +1,43 @@
+<div layout="column" layout-margin style="padding:25px;">
+  <h2 class="md-title">Determinate</h2>
+
+  <p>For operations where the percentage of the operation completed can be determined, use a determinate indicator. They
+     give users a quick sense of how long an operation will take.</p>
+  <div layout="row" layout-sm="column" layout-align="space-around">
+    <md-progress-circular mode="determinate" [value]="determinateValue"></md-progress-circular>
+  </div>
+
+  <h2 class="md-title">Indeterminate</h2>
+
+  <p>For operations where the user is asked to wait a moment while something finishes up, and it's not necessary to
+     expose what's happening behind the scenes and how long it will take, use an indeterminate indicator.</p>
+  <div layout="row" layout-sm="column" layout-align="space-around">
+    <md-progress-circular mode="indeterminate"></md-progress-circular>
+  </div>
+
+  <h2 class="md-title">Indeterminate</h2>
+
+  <p>
+    Your current theme colors can be used to easily colorize your progress indicator with `md-warn` or `md-accent`
```

Comment:
The examples don't support markdown, perhaps wrap the `md-warn` and `md-accent` notes in `<code></code>` blocks?